src/style.css

```diff
@@ -1 +1,86 @@
+/* stylelint-disable property-no-unknown */
 /* Styles go here */
+body {
+  padding: 0;
```

Comment:
body doesn't have paddings. It's senseless to zero them. You need to zero only margins.